compareECLFiles.cmake

```diff
@@ -399,6 +399,13 @@ add_test_compareECLFiles(CASENAME udq_wconprod
                          REL_TOL ${rel_tol}
                          DIR udq_actionx)
 
+add_test_compareECLFiles(CASENAME udq_pyaction
```

Comment:
this needs to be wrapped in if(OPM_ENABLE_PYTHON)

Reply:
Done

Reply:
The relevant cmake flag - `OPM_ENABLE_EMBEDDED_PYTHON` is defined and set in opm-common; seems it is not available here? The test is never run.

Reply:
right. in that case it needs to be published through the opm-common-config.cmake.

Reply:
Ehh - exactly which file is this?